test/core/watch/WatchViewModelGetNodeLabelTree.dyn

```diff
@@ -0,0 +1,219 @@
+{
+  "Uuid": "ceace55e-ec4b-4f69-94e9-9b7286a2babd",
+  "IsCustomNode": false,
+  "Description": null,
+  "Name": "WatchViewModelGetNodeLabelTree",
+  "ElementResolver": {
```

Comment:
I know this is probably more or less the same as the description gif, but its useful to have an image of the test dyn file in the PR description, makes it easier to understand what is going on. can we add that?

Reply:
![image](https://user-images.githubusercontent.com/193290/101066475-5e615f80-358e-11eb-99cc-666ab5d51caa.png)